string project_name

# 0: initial value; 1: shutdown; 2: reboot; 3: cleanup server process
int8 system_cmd

nodeParams node_params
